.eslintrc.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     // node files
     {
       files: [
-        'index.js',
+        './index.js',
```

Comment:
I do not think this works actually. This will definitely make it so that this rule does not cause `addon/index.js` to be linted as if it were node, but it no longer matches the root level `index.js` itself.  Coincidentally, I am working on fixing this upstream in eslint (https://github.com/eslint/eslint/issues/9740).

The fix here (for now) is to leave this entry as `'index.js'`, but add `excludedFiles: ['addon/**']` just below `files`.

Reply:
ah, thanks for the explanation!